src/main/java/io/github/noeppi_noeppi/libx/crafting/ingredient/NbtIngredient.java

```diff
@@ -2,17 +2,21 @@
 
 import com.google.gson.JsonElement;
 import com.google.gson.JsonObject;
+import io.github.noeppi_noeppi.libx.annotation.meta.RemoveIn;
 import net.minecraft.nbt.CompoundTag;
 import net.minecraft.network.FriendlyByteBuf;
 import net.minecraft.world.item.ItemStack;
 import net.minecraft.world.item.crafting.Ingredient;
 import net.minecraftforge.common.crafting.CraftingHelper;
 import net.minecraftforge.common.crafting.IIngredientSerializer;
+import net.minecraftforge.common.crafting.NBTIngredient;
 
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 import java.util.stream.Stream;
 
+// TODO rename in 1.19 and use share tag by default because ingredients are
+//  synced with the client.
```

Comment:
Only if you’re editing anything else, please remove the one space at beginning.

Reply:
The one space at the beginning actually serves a purpose. Because of this space, IntelliJ treats both lines as one `TODO` comment and shows both of them in the TODO-View.

Reply:
Ahh good to know, ignore it :)